python/ray/tests/test_advanced_2.py

```diff
@@ -580,6 +580,14 @@ def f():
     ray.get(results)
 
 
+def __delete_miscellaneous_item__(resources):
```

Comment:
Sorry. Can you just move the method inside `test_zero_capacity...` function and name it without prepending / appending `__`?